Source/Core/Gen4/Searchers/WildSearcher4.cpp

```diff
@@ -546,7 +546,7 @@ std::vector<WildState> WildSearcher4::normalMethodK(WildState state, u32 seed) c
     {
         if ((nextRNG % 25) == state.getNature())
         {
-            state.setLead(Lead::None);
+            lead == Lead::CompoundEyes? state.setLead(Lead::CompoundEyes) : state.setLead(Lead::None);
```

Comment:
Probably just be better to do `state.setLead(lead)`

Reply:
done